src/pal/src/exception/remote-unwind.cpp

```diff
@@ -764,6 +777,57 @@ ExtractProcInfoFromFde(const libunwindInfo* info, unw_word_t* addrp, unw_proc_in
     return true;
 }
 
+static void UnwindContextToContext(unw_cursor_t *cursor, CONTEXT *winContext)
```

Comment:
Why do we need to have a separate copy of this function instead of sharing the one in the seh-unwind.cpp?

Reply:
For some reason that I can't figure out, calling the function in seh-unwind.cpp doesn't work. It might be because of the UNW_LOCAL_ONLY define at the top of the file. Maybe it messes with the unw_get_reg macro/function.

Reply:
Ah, right, that's likely the reason.